src/Microsoft.AspNet.Hosting.Interfaces/HostingEnvironmentExtensions.cs

```diff
@@ -9,6 +9,29 @@ namespace Microsoft.AspNet.Hosting
 {
     public static class HostingEnvironmentExtensions
     {
+        private const string DevelopmentEnvironmentName = "Development";
```

Comment:
Should we put `public static readonly` versions of these somewhere? It would probably be good for devs to be able to access the "official" Dev and Production names.